robot_sort/robot_sort.py

```diff
@@ -96,8 +96,48 @@ def sort(self):
         """
         Sort the robot's list.
         """
-        # Fill this out
-        pass
+        while self.light_is_on() == False:
+            self.set_light_on()
+
+            if self.can_move_left() == False:
+                while self.can_move_right() == True:
+                    if self.compare_item() == None:
+                        self.swap_item()
+                        self.move_right()
+                    if self.compare_item() == 1:
+                        self.swap_item()
+                        self.move_left()
+                        self.swap_item()
+                        self.move_right()
+                    
```

Comment:
Nice job on the robo-sort overall. You recognized that you could use the light as a Boolean and developed a pretty good algorithm for solving the challenge.

There are some other ways we could go about solving this that will take less "time" for the robot. Maybe this above snippet is causing a few extra moves... perhaps we could develop a method that does not have to move backwards every time it needs to make a swap. Perhaps 'None' could be used as a placeholder object to make up for us not being able to index items.